pkgs/development/python-modules/jsonfield/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchPypi, buildPythonPackage, django, pytest, pytest-django }:
+
+buildPythonPackage rec {
+  pname = "jsonfield";
+  version = "3.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yl828cd0m8jsyr4di6hcjdqmi31ijh5vk57mbpfl7p2gmcq8kky";
+  };
+
+  checkInputs = [ pytest pytest-django ];
+
+  checkPhase = ''
+    PYTHONPATH="$(pwd):$PYTHONPATH" \
```

Comment:
Does it any harm to export these?

Reply:
I think I've found a way to use pytestCheckHook by exporting just the DJANGO_SETTING_MODULE in the preCheck phase

Reply:
https://github.com/NixOS/nixpkgs/pull/107536/commits/3ef93692b5a360be8575e242bf833fdc15472a55